assets/check

```diff
@@ -70,16 +70,12 @@ fi
 
 if [ -n "$tag_filter" ]; then
   {
-    if [ -n "$ref" ]; then
-      git tag --list "$tag_filter" --sort=creatordate --contains $ref
-    else
-      git tag --list "$tag_filter" --sort=creatordate | tail -1
-    fi
+    git tag --list "$tag_filter" --sort=creatordate | tail -1
```

Comment:
I might see the error you were having.

The tags that get detected are only tags that *purposely* have commits off of a specific branch. If you were missing a tag that means it was a SHA that did not exist in the `branch` configured. This is to ensure that consistent versioning is maintained with concourse and SHAs it knows about.

This PR looks for all tags, which would include tags that weren't made against consistent branch.

@Wintermute1 can you confirm the tag that was missing was not made on a SHA that exists in your configured `branch` or the default `master`?